test/integration/environment_test.go

```diff
@@ -0,0 +1,91 @@
+//go:build integration
+// +build integration
+
+package integration
+
+import (
+	"fmt"
+	"reflect"
+	"testing"
+
+	"github.com/aptible/go-deploy/aptible"
+)
+
+func TestEnvironments(t *testing.T) {
+	// Setup
+	orgID := getOrgID(t)
+	stackID := getStackID(t)
+	client := getClient(t)
+	var environment aptible.Environment
+	var err error
+
+	// Test
+	testNotFound := func(id int64) {
+		environment, err = client.GetEnvironment(id)
+		if err != nil {
+			t.Error("Expected GetEnvironment to not return an error but got", err.Error())
+		}
+	}
```

Comment:
Need to check if `environment.Deleted == true` to ensure we got back a 404 instead of actually finding an existing environment.

Reply:
i did do that test below, this one includes ones that are just straight up 404s

Reply:
You're right.